authentication/views.py

```diff
@@ -25,18 +23,14 @@ def post(self, request):
         form = SignUpForm(request.POST)
         print(form.is_valid())
         if form.is_valid():
-
-            form_data = form.clean()
-            new_user_data = {
-                    'username': form_data.get('username'),
-                    'email': form_data.get('email'),
-                    'password': form_data.get('password')
-            }
             try:
-                User.objects.create_user(new_user_data['username'], new_user_data['email'], new_user_data['password'])
+                form.email_clean()
```

Comment:
This statement is not needed since form's `is_valid` calls `clean` also.

Reply:
removed.